pkg/network/sdn-cni-plugin/openshift-sdn.go

```diff
@@ -230,6 +230,22 @@ func (p *cniPlugin) CmdAdd(args *skel.CmdArgs) error {
 			if err := netlink.RouteAdd(route); err != nil {
 				return fmt.Errorf("failed to add route to service network: %v", err)
 			}
+
+			// Add a route to DNS IP via SDN
```

Comment:
should probably have a loop for this and the node IP and the service CIDR rather than having basically the same code three times. (If you just add "/32" to the end of the two IPs you can treat them all as CIDRs).

(And I think it does make sense to keep the node and service special cases, even with the DNS IP being handled explicitly, because there might be other reasons besides DNS that we care about access to those IPs in the future, and they are only available via the SDN interface.)

Reply:
Yes, that was the same reason for keeping node IP and service CIDR routes.
Okay, I will create a loop for this.